### ISSUE TITLE
d3dx9_43.dll not found

### ISSUE THREAD

**Issue by ghost** [brian-pantano/PianoFromAbove#15]
I have downloaded Piano From Above, and I have gone to launch it and get displayed with the screen to enter the admin password. Because I don't know the admin password, I extracted the file and found the file without the admin sign thing. I go click on it i get displayed with a message that says "The code execution cannot proceed because d3dx_43.dll was not found. Reinstalling the program may fix this problem." I have reinstalling it and got the error, is there way I can get around this error, or get a version that doesn't require the admin password.
Cheers.

**Comment by OSToastBach** [comment]
Yes hello there, you can try to reinstall directx9 or you can find d3dx9_43.dll online and then install them in your system32 and syswow64 directory, if you do not want to put them there then you can also put it inside the directory where the PFA EXE is and it should work

**Comment by blulere** [comment]
You can go to dll-files.com and search for the file name (or just search for d3dx9_43.dll and extract it into the PFA directory. That might work
